code/_helpers/text.dm

```diff
@@ -50,6 +50,9 @@
 
 	return input
 
+/proc/sanitizeFileName(var/input)
```

Comment:
Did you really need to create this? Surely sanitising functions already exist?

Reply:
Yes, its for file names, there is no analog